### ISSUE TITLE
[KIEKER-1304] Add PMD rules from TeeTime

### ISSUE THREAD

**Issue by rju** [kieker-monitoring/kieker#2438]
JIRA Issue: [KIEKER-1304] Add PMD rules from TeeTime
Original Reporter: Christian Wulf
 *** 
<p>&#45; MissingPluralVariableName
<br class="atl-forced-newline" />- MissingUnitSuffix
<br class="atl-forced-newline" />- NoEmptyLineAfterMethodHeader
<br class="atl-forced-newline" />- NonHeaderCommentSize (replaces PMD's CommentSize rule)
<br class="atl-forced-newline" />- TypeJavadocShouldStartWith </p>

<p>Including
<br class="atl-forced-newline" />- Eclipse plugin fragment including these rules</p>

[KIEKER-1304]: https://kieker-monitoring.atlassian.net/browse/KIEKER-1304?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by rju** [comment]
author André van Hoorn -- Thu, 3 May 2018 11:24:43 +0200
<p>What is the status of this ticket?</p>

**Comment by rju** [comment]
author Unknown -- Fri, 4 May 2018 23:40:07 +0200
<p>I have updated the corresponding branch. The custom rules are now in PMD's classpath. They are already activated via the config file "pmdrules.xml" (search for ``&lt;rule ref="rulesets/chw/basic.xml"&gt;`` in the file). Since the additional rules cause additional violations, I have increased the thresholds. See the link to the corresponding PR on the right side.</p>
